arangod/Agency/Supervision.cpp

```diff
@@ -851,8 +867,8 @@ void Supervision::run() {
     _agent->executeLockedRead([&]() {
       if (_agent->readDB().has(supervisionNode)) {
         try {
-          _snapshot = _agent->readDB().get(supervisionNode);
-          if (_snapshot.children().size() > 0) {
+          auto const sn = _agent->readDB().get(supervisionNode);
```

Comment:
This is a change in behaviour. _snapshot is no longer initialized here. Is this safe?